### ISSUE TITLE
Option to disable Alt Key navigation

### ISSUE THREAD

**Issue by AgamlaRage** [sindrets/atom-title-bar-replacer#49]
Hello, 

thx for this great package!

### What I want
I would like to totally disable the `Alt Key Navigation` to be able to bind my own keybindings starting with `Alt`.
Could it be possible to add an options to the settings to achieve this ?

### Related issues
As I understand this is a default behavior:
- #2
- #6 
- #12

### Example of one of my custom `Alt Keybinding` that is not working with `title-bar-replacer`
```cson
# keymap.cson
'atom-workspace atom-text-editor:not([mini])':
  'alt-right': 'editor:move-to-end-of-screen-line' # go to end of line
```

**Comment by matthewgdv** [comment]
Hi, seconding this. Please add an option to disable menu bar navigation using alt. It's breaking existing alt-left and alt-right keybindings.

**Comment by sindrets** [comment]
Fixed in v2.0.0. There's a setting to disable both "Alt Gives Focus" and "Menu Bar Mnemonics". But I have also improved the keyboard navigation, so it is now possible to map key bindings with alt without disabling the alt key navigation.
